src/components/Stakeholders/StakeholderDetail/Header.js

```diff
@@ -149,6 +153,10 @@ export default function StakeholderDialogHeader({ data }) {
 
   const defaultTypeList = ['Organization', 'Person'];
 
+  useEffect(() => {
+    if (forceSave) handleSave();
+  });
```

Comment:
To prevent unnecessary calls, pass `[forceSave]` as a second parameter to `useEffect()`.

Reply:
I have updated this one